### PR TITLE
[feat] add multiple tabs support

### PR DESCRIPTION
**Pull request by guilhermeljs**

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -40,11 +44,32 @@
   function handleUp() {
     navigation.up();
   }
+
+  function handleNewTab(){
+    tabNavigation.create("C:/");
```
Comment (jordyfontoura) [inline src/App.svelte:49]: Consegue mudar esse cara `C:/` para `homedir`?
Tem uma função disso no rust, podemos até colocar ele em um context para n precisar ficar consultando.

File: src/App.svelte [hunk 2]
```diff
@@ -40,11 +44,32 @@
...
+  function handleNavigateTab(id: number){
+    tabNavigation.navigate(id)
+  }
+
 </script>
 
 <div class="container">
+  <div class="tabs">
+    {#each $sessions.sessions as tab}
+      <button on:click={()=>handleNavigateTab(tab.id)} class={tab.current ? "tab-active": "up"}>{tab.id}</button>
```
Comment (jordyfontoura) [inline src/App.svelte:66]:
```svelte
 <button on:click={()=>handleNavigateTab(tab.id)} class="tab" class:active={tab.current}>{tab.id}</button>
```
Wouldn't something like this be better?
Don't forget to change "tab-active" to ".tab.active" in css

File: src/styles.css [hunk 3]
```diff
@@ -82,6 +82,15 @@
   animation: fileFocus 4s infinite;
 }
 
+.tabs {
+  width: 100%;
+  display: flex;
+}
+
+.tab-active {
```
Comment (jordyfontoura) [inline src/styles.css:90]: Please change `.tab-active` to `.tab.active` as mention at https://github.com/jordyfontoura/wbe/pull/5/files#r1505140120

File: src/App.svelte (hunk 1 above)
Comment (guilhermeljs) [reply src/App.svelte:49]: Actually, we can just pass null, and it will use the homedir. I'm gonna update right now.

File: src/App.svelte (hunk 2 above)
Comment (guilhermeljs) [reply src/App.svelte:66]: Absolutely!

**Review by jordyfontoura — APPROVED** [review]

**Pull request merged**